tools/make/kube.mk

```diff
@@ -101,4 +103,20 @@ delete-cluster: $(tools/kind) ## Delete kind cluster.
 
 .PHONY: release-manifests
 release-manifests: $(tools/kustomize) ## Generate Kubernetes release manifests.
-	tools/hack/release-manifests.sh $(GATEWAY_API_VERSION) $(TAG)
+	@mkdir -p release-artifacts/
+	@curl -sLo release-artifacts/gatewayapi-crds.yaml ${GATEWAY_RELEASE_URL}
+	@echo "Added: release-artifacts/gatewayapi-crds.yaml"
+
+	@mkdir -pv $(OUTPUT_DIR)/manifests/provider
```

Comment:
can you try and reuse the previous work you did in https://github.com/envoyproxy/gateway/blob/2fb3ca8954705dabbcfa08007b658655fcb60f46/tools/make/kube.mk#L41

Reply:
That will apply the outputs, but right, we can make this more common used.